nornir/core/connections.py

```diff
@@ -18,12 +18,7 @@ class ConnectionPlugin(ABC):
 
     __slots__ = ("connection", "state")
```

Comment:
how is it possible this works without updating `__slots__`?

Reply:
Yeah, don't know...would have to dig into that :-)